tests/gpu-pod.yaml

```diff
@@ -7,7 +7,7 @@ spec:
   containers:
     - name: cuda-vector-add
       # https://catalog.ngc.nvidia.com/orgs/nvidia/teams/k8s/containers/cuda-sample
-      image: "nvcr.io/nvidia/k8s/cuda-sample:vectoradd-cuda11.7.1-ubuntu20.04"
+      image: "nvcr.io/nvidia/k8s/cuda-sample:vectoradd-cuda12.5.0"
```

Comment:
Why was this change included in this PR?

Reply:
See [here](https://github.com/NVIDIA/gpu-operator/pull/937/files#r1722645761)